### DISCUSSION TITLE
Add in which group/s is the package included

### DISCUSSION THREAD

**Discussion by mrodespin** [vtsykun/packeton#125] [Ideas]
It could be nice to have in the package detail page where we actually have the repo url, number of installs, dependents, etc a new section specifying in which group/s the package is included

**Comment by vtsykun** [comment]
Done
![Selection_1543](https://github.com/vtsykun/packeton/assets/21358010/72a0e518-0697-4ea9-88b2-498df6266b2a)

**Reply by mrodespin** [reply]
Thanks for your celerity @vtsykun!
